x-pack/plugins/triggers_actions_ui/public/application/components/add_message_variables.tsx

```diff
@@ -22,17 +30,30 @@ export const AddMessageVariables: React.FunctionComponent<Props> = ({
   const [isVariablesPopoverOpen, setIsVariablesPopoverOpen] = useState<boolean>(false);
 
   const getMessageVariables = () =>
-    messageVariables?.map((variable: string, i: number) => (
+    messageVariables?.map((variable: ActionVariable, i: number) => (
       <EuiContextMenuItem
-        key={variable}
+        key={variable.name}
         data-test-subj={`variableMenuButton-${i}`}
         icon="empty"
         onClick={() => {
-          onSelectEventHandler(variable);
+          onSelectEventHandler(variable.name);
           setIsVariablesPopoverOpen(false);
         }}
       >
-        {`{{${variable}}}`}
+        <>
+          <EuiText size="m">{`{{${variable.name}}}`}</EuiText>
+          <EuiText size="m" color="subdued">
+            <div className="euiTextColor--subdued">
+              <FormattedMessage
+                id="xpack.triggersActionsUI.components.addMessageVariable.variableDescriptionLabel"
+                defaultMessage="{description}"
```

Comment:
Same here in regards to the difference between calling FormattedMessage and simply rendering `{description}`.